docs/event.md

```diff
@@ -120,6 +123,8 @@ Allows you to test if your webserver responds to subscription requests properly.
 | `streamdown`        | Stream offline event.                                         |
 | `add-moderator`     | Channel moderator add event.                                                                               |
 | `remove-moderator`  | Channel moderator removal event.                                                                           |
+| `ban`       		  | Channel ban event.                                                                           			   |
```

Comment:
The pipes don't line up if we're being really strict here ;)

Reply:
ha! i can fix that actually.